include/wx/qt/window.h

```diff
@@ -30,6 +30,7 @@ class QEvent;
 class QCloseEvent;
 class QContextMenuEvent;
 class QFocusEvent;
+class QString;
```

Comment:
This is useless when passing `QString` by value as is done here, forward declaration is not enough for this anyhow.